modern/src/App.js

```diff
@@ -23,9 +23,19 @@ async function getSkin() {
 }
 
 function handleMouseEventDispatch(node, event, eventName) {
-  const rect = event.target.getBoundingClientRect();
-  const x = event.clientX - rect.left;
-  const y = event.clientY - rect.top;
+  event.stopPropagation();
+
+  // In order to properly calculate the x/y coordinates like MAKI does we need
+  // to find the top level absolute positioned element and calculate based off of that
+  let relativeParent = event.target.offsetParent;
+  while (
+    relativeParent.offsetParent &&
+    relativeParent.offsetParent !== document.body
+  ) {
+    relativeParent = relativeParent.offsetParent;
+  }
+  const x = event.clientX - relativeParent.offsetLeft;
+  const y = event.clientY - relativeParent.offsetTop;
```

Comment:
I was imagining we would do this by traversing up our own state tree (Maki or Redux) until we find a container, and then using the x/y values that we currently have from the last time we set them.

Reply:
oh yea, thats much much better!

Reply:
yea, just fixed it...SO MUCH BETTER!!!